mir_perception/mir_handle_detection/ros/include/mir_handle_detection/drawer_handle_perceiver.h

```diff
@@ -45,6 +46,7 @@ class DrawerHandlePerceiver
         bool is_running;
         int retry_attempts_;
         int num_of_retries_;
+        geometry_msgs::PoseStamped pose_stamped;
```

Comment:
hmm, actually I think it would be a bit clearer what is going on if you pass `pose_stamped` as an argument to `extractPlaneOutlier` (instead of making it a member variable)

Reply:
Done